doc_admin/AdminManual.adoc

```diff
@@ -57,11 +61,11 @@ The aim of this program is to manage, in the simplest manner, the hospital admin
 
 Open Hospital is a stand-alone Javafootnote:[Java is a registered trademark of Oracle and/or its affiliates.] software program that does not require installation and is often known as a portable installation. The .zip file can be copied and moved everywhere on the target system, whether the operating system is Windows or Linux.
 
-The software acts as a client application. The server part utilizes a DBMS (Database Management System) to persist information. From the inception of the software, the free and open-source MySQLfootnode:[See https://www.mysql.com/] database system is the DBMS of choice.
+The software acts as a client application. The server part utilizes a DataBase Management System (DBMS) to archive, share and present the information. The free and open-source MySQLfootnode:[See https://www.mariadb.org] MariaDB is the database server of choice.
```

Comment:
should `open-source MySQLfootnode:[See https://www.mariadb.org] MariaDB is the database server of choice.` rather be: `open-source MariaDBfootnode:[See https://www.mariadb.org] is the database server of choice.`?

Reply:
fixed